### DISCUSSION TITLE
In which direction does the raspberry pi zero 2 go?

### DISCUSSION THREAD

**Discussion by MaddoScientisto** [frntc/Sidekick64#68] [Q&A] [answered]
I have a pi zero 2 without headers soldered on yet and before I solder them I realized I have no idea which direction the pi zero 2 is supposed to be mounted in relative to the board. Frustratingly there are no photos of the underside of the board around.

Here are some options, please help me figure out which one is correct.

Option A: 
![image](https://user-images.githubusercontent.com/816014/229289969-2e83c5da-1e5f-455a-a28c-a6fced0c9b18.png)

Option B:
![image](https://user-images.githubusercontent.com/816014/229289975-92449894-4582-494d-ac5c-3dbd1778aec2.png)

Option C:
![image](https://user-images.githubusercontent.com/816014/229289980-aad74ec8-6104-4604-a1ec-e891d9a39408.png)

Option D:
![image](https://user-images.githubusercontent.com/816014/229289987-9b2dbd27-a9df-4e29-aec4-513dfc944d37.png)

I only have either these long passthrough headers or a short one, I'm afraid the short one may be too short 
![image](https://user-images.githubusercontent.com/816014/229289991-6ab85c55-1453-444f-84a8-63526d37282c.png)

**Comment by StmLord** [comment, marked as answer]
first picture is the correct orientation. There is a mark though for pin 1 both on rpi and on sk

**Reply by MaddoScientisto** [reply]
Thank you!
